GettingStarted.md

```diff
@@ -108,16 +107,17 @@ and that it follows one of the formats at https://github.com/sailfishos/rpmlint/
 ## Other useful notes
 
 * You can create sub-projects in your home project to group builds together.
-* You can add the repository for the builds to your device using `zypper ar` to install
+* You can add a repository to your device using `ssu ar` in order to install your built packages 
   on your device.
 * To re-trigger OBS to download the latest code, click "Trigger services" at the package overview.
 * You can configure OBS to not build certain packages against certain repositories, or disable
   certain projects from being published.  It is very flexible.  This is available from the "Repositories"
   tab.
-* OBS will resolve dependencies during the build, if packages are available in either the current project or 
-  the build target.  If you want to build a package in your home project against other packages, which are only
-  available in SailfishOS:Chum (or another repository), then these can be added to the repository configuration
-  with code as the following in the project "Meta" configuration:
+* OBS will resolve dependencies when building a package, if dependency packages are available in 
```

Comment:
, is unnecessary here

Reply:
Well, yes, but …
* the comma is not introduced or altered by this commit.
* IMO the comma increases the readability of this lengthy sentence.

Do you feel / think it has to go?